paasta_tools/api/api_docs/swagger.json

```diff
@@ -1919,10 +2094,6 @@
             "type": "object",
             "description": "Kafka instance metadata"
         },
-        "InstanceSpecKafkaCluster": {
```

Comment:
Was this intentionally removed?

Reply:
Yes, it was. According to the openapi-generator @ 194e02b
```
Validating spec (paasta_tools/api/api_docs/swagger.json)
Warnings:
	- Unused model: InstanceSpecKafkaCluster
```